### ISSUE TITLE
Migrate from jboss-openjdk base images to RHEL images

### ISSUE THREAD

**Issue by kyguy** [jboss-container-images/amqstreams-1-openshift-image#59]
Cloud Core openjdk images are being deprecated in favor for rhel images: https://issues.jboss.org/browse/CLOUD-2967 to simplify builds

**Comment by scholzj** [comment]
Does it mean that we should do our own Java base image from RHEL?

**Comment by kyguy** [comment]
Yes, I believe cloud core is going to make an openjdk module we can install ontop of the RHEL image, giving us the same functionality as the openjdk image without the respin pain. Next release we are going to have to switch to a RHEL base image

**Comment by ppatierno** [comment]
More or less it's the same we do upstream using CentOS as base image and building our own Java base.
